openshift/ci-operator/generate-ci-config.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
From what repo did you take this file? It is different than eventing-kafka.

Reply:
@aliok we customize this file for each repo based on which images it needs and how should the ci-operator end result YAML should look like.